test/integration/http.js

```diff
@@ -393,15 +393,79 @@ describe('HTTP APIs', function () {
         .expect(hasHeader('acl', 'baz.ttl' + suffixAcl))
         .expect(201, done)
     })
-    it('should return 409 code when trying to put to a container',
-      function (done) {
-        server.put('/')
-          .expect(409, done)
-      }
-    )
-    // Cleanup
-    after(function () {
-      rm('/foo/')
+
+    describe('PUT and containers', () => {
+      const containerMeta = fs.readFileSync(path.join(__dirname,
+        '../resources/sampleContainer/post2.ttl'),
+        { 'encoding': 'utf8' })
+
+      before(() => {
+        rm('/foo/')
+      })
+
+      after(() => {
+        rm('/foo/')
+      })
+
+      it('should create a container (implicit from url)', () => {
+        return server.put('/foo/two/')
+          .expect(201)
+          .then(() => {
+            let stats = fs.statSync(path.join(__dirname, '../resources/foo/two/'))
+
+            if (!stats.isDirectory()) {
```

Comment:
You can just write `expect(statsisDirectory()).toBe(true)` if you're using the `expect` library.